components/log-viewer-webui/common/searchResultsMetadata.ts

```diff
@@ -24,10 +21,12 @@ enum SEARCH_SIGNAL {
  */
 interface SearchResultsMetadataDocument {
     _id: string;
-    errorMsg: Nullable<string>;
+    errorMsg: string | null;
```

Comment:
it is equivalent to `Nullable<string>`?
```suggestion
    errorMsg: Nullable<string>;
```